### ISSUE TITLE
Thinking about 'sync_batchnorm.batchnorm.convert_model(module)'..

### ISSUE THREAD

**Issue by chrisway613** [vacancy/Synchronized-BatchNorm-PyTorch#35]
I've taken a close look at the source code in this method, and I have thought about the code below:

line 1: if isinstance(module, torch.nn.DataParallel):
line 2:        mod = module.module
line 3:        mod = convert_model(mod)
line 4:        mod = DataParallelWithCallback(mod)
line 5:        return mod

Whether line 4 should be 'DataParallelWithCallback(mod, device_ids=module.device_ids)'?
this can keep the same cuda device ids, otherwise, it'll use all cuda devices which is available.

Please feel free to comment, and show your thoughts, thanks!

**Comment by vacancy** [comment]
Fixed by 2351227. Thank you!
